_glaredb/examples.md

```diff
@@ -0,0 +1,187 @@
+
+They need a solution that allows them to query across all these data sources in a unified manner, using the SQL language that their team is familiar with. This is where GlareDB comes in. It allows creating external tables from different data sources, making it possible to execute SQL queries across all these diverse data storage systems. This solution will significantly streamline their data analysis process and generate insights more effectively.
+
+## Step 1: MongoDB - Customer Data
+
+In MongoDB, you have a `customers` collection with documents structured as follows:
+
+```json
+{
+  "_id": {
+    "oid": "507f191e810c19729de860ea"
+  },
+  "first_name": "John",
+  "last_name": "Doe",
+  "email": "john.doe@example.com",
+  "location": "United States",
+  "membership_status": "premium",
+  "signup_date": {
+    "date": "2018-07-14T00:00:00Z"
+  }
+}
+```
+
+## Step 2: Postgres - Transactional Data
+
+In your Postgres database, you have a `transactions` table structured as follows:
+
+```sql
+CREATE TABLE transactions (
+  transaction_id INT PRIMARY KEY,
+  customer_id INT,
+  product_id INT,
+  purchase_date TIMESTAMP,
+  amount DECIMAL(10, 2)
+);
+```
+
+## Step 3: AWS S3 - Unstructured Customer Review Data
+
+On AWS S3, you store unstructured data in the form of JSON files for customer reviews. Here's an example:
+
+```json
+{
+  "review_id": "R1",
+  "customer_id": "507f191e810c19729de860ea",
+  "product_id": "P100",
+  "review_date": "2022-05-14",
+  "rating": 5,
+  "comment": "Great product, fast delivery. Highly recommend!"
+}
+```
+
+## Step 4: GCP - Predictive Analytics Output
+
+On Google Cloud Platform, you store CSV files that represent the output from predictive models, such as next purchase prediction:
+
+```csv
+customer_id,next_purchase_product_id,next_purchase_date
+507f191e810c19729de860ea,P101,2023-02-01
+```
+
+## Step 5: Creating External Tables with Our Tool
+
+With GlareDB, you can create external tables linked to your diverse data sources. Here's how you would define the tables for each source:
+
+**MongoDB (Customer Data)**
+
+```sql
+CREATE EXTERNAL TABLE customers
+FROM mongodb
+OPTIONS (
+    connection_string = '<connection-string>',
+    database = 'default',
+    collection = 'customers'
+);
+```
+
+**Postgres (Transactional Data)**
+
+```sql
+CREATE EXTERNAL TABLE transactions
+FROM postgres
+OPTIONS (
+    host = 'my.postgres.host',
+    port = '5432',
+    user = 'user',
+    password = 'password',
+    database = 'default',
+    schema = 'public',
+    table = 'transactions'
+);
+```
+
+**AWS S3 (Unstructured Customer Review Data)**
+
+```sql
+CREATE EXTERNAL TABLE reviews
+FROM s3
+OPTIONS (
+    region = 'us-west-2',
+    access_key_id = 'my_access_key_id',
+    secret_access_key = 'my_secret_access_key',
+    bucket = 'my_bucket',
+    location = 'reviews/*.json'
```

Comment:
Still needs to be added in (cc @vrongmeal )

Reply:
looks like this was merged? 
https://github.com/GlareDB/glaredb/pull/1410

Reply:
Glob support for scan functions exists but there's an open PR for adding the same to external table. Should be in by tomorrow. (https://github.com/GlareDB/glaredb/pull/1436)